### DISCUSSION TITLE
Turning off autopilot on Appr just before flare behaves differently now, elevator control

### DISCUSSION THREAD

**Discussion by BozoBBozo** [flybywiresim/aircraft#2997] [Help] [answered]
I'm using today's (Jan 11) Dev build, but this also occurred with the previous build, but has NOT happened in dozens of A320 landings in all conditions over the past couple of months. I'm using my same ILS approach techniques, same controller, I've been using all along.

The symptom: ILS approach, on localizer/glideslope in Appr mode. Everything going perfectly. As is my practice (and not uncommon, I think) right about the 100 AGL mark I switch off the autopilot and finish flaring/landing myself. Normally this works just fine; when switching off A/P the aircraft continues to behave and control normally. But now, it seems like the aircraft is way out of trim (just a guess); as soon as I turn off A/P the nose dives towards the runway (which at < 100' is quite the excitement). This behavior has just started in builds over the past 2-5 days.

Is this a "just me" thing? If not, is it intentional behavior (I can't imagine why, but I have to ask)?

**Comment by OT1282** [comment]
I have the same problem, whether I land with or without autoland.  Also, today I saw that there was an update to the dev version and downloaded it.  As soon as I finished the flight, there was another notice of an update.  I did it and flew again and noticed no difference.  I deleted the current one, rebooted my computer, downloaded again and immediately restarted the installer and it said there was another update.  I deleted and installed the stable version...  Tomorrow is another day..

**Comment by BozoBBozo** [comment]
I reverted to Stable 0.5.2 and completed a ~300nm flight and followed my usual procedure. No pitch-down issues when turning off A/P at about 100 feet AGL.

**Comment by hotshotp** [comment, marked as answer]
it has always been like this for the past like 3 months - will be fixed with custom flybywire
